specification/compute/resource-manager/Microsoft.Compute/2017-10-15-preview/swagger/computeArtifactPublishingService.json

```diff
@@ -0,0 +1,870 @@
+{
+  "swagger": "2.0",
```

Comment:
Not related to swagger - but please make sure these new options are added to the /operations API that compute owns.